pipeline/extract_epic.py

```diff
@@ -0,0 +1,133 @@
+"""Extracts script that pulls game data from undocumented GraphQL API"""
+
+import requests
+
+
+def extract_games(url: str):
+    """Queries a graphQL API """
+    q = """{
+  Catalog {
+    searchStore(
+      allowCountries: "GB"
+      category: "games/edition/base"
+      comingSoon: false
+      count: 50
+      country: "GB"
+      locale: "en-US"
+      sortBy: "releaseDate"
+      sortDir: "DESC"
+    ) {
+      elements {
+        id
+        title
+        description
+        currentPrice
+        releaseDate
+        developerDisplayName
+        publisherDisplayName
+
+        seller {
+          id
+          name
+        }
+
+        keyImages {
+          type
+          url
+          alt
+        }
+
+        items {
+          id
+          namespace
+        }
+
+        tags {
+          id
+          name
+          groupName
+        }
+        
+        price(country: "GB") {
+          totalPrice {
+            discountPrice
+            originalPrice
+            voucherDiscount
+            discount
+            currencyCode
+            currencyInfo {
+              decimals
+            }
+            fmtPrice {
+              originalPrice
+              discountPrice
+              intermediatePrice
+            }
+          }
+        }
+      }
+    }
+  }
+}"""
+
+    response = requests.post(url=url, json={"query": q})
+    print("Response status code:", response.status_code)
+
+    if response.status_code == 200:
+        return response.json()
+
+    return None
+
+if __name__ == "__main__":
+
+    data = extract_games("https://graphql.epicgames.com/graphql")
+
+    ## How to extract elements from data!
+    if data:
+        games = data["data"]["Catalog"]["searchStore"]["elements"]
+        for game in games:
+            print(f"Title: {game['title']}")
+            print(f"Release Date: {game['releaseDate']}")
+            print(f"Developer: {game['developerDisplayName']}")
+            print(f"Publisher: {game['publisherDisplayName']}")
+            print(f"Seller: {game['seller']['name']}")
+            print(f'Current Price: {game['currentPrice']}')
```

Comment:
```suggestion
            print(f"Current Price: {game['currentPrice']}")
```

extract_epic.py:94:43: E0001: Parsing failed: 'f-string: unmatched '[' (extract_epic, line 94)' (syntax-error)